corehq/apps/accounting/models.py

```diff
@@ -785,9 +785,12 @@ class Meta(object):
         unique_together = ('edition', 'is_trial', 'is_report_builder_enabled')
 
     @classmethod
+    @quickcache(['edition', 'is_trial', 'is_report_builder_enabled'], timeout=24 * 60 * 60)
```

Comment:
If this is cached for 24 hours it will take that long before a new version becomes active. Is that likely to be an issue? I'm not sure how often new default plans are created or quite what the process is.

Reply:
default plans are created only rarely, via a db migration.  I believe this was the most recent change:
https://github.com/dimagi/commcare-hq/blob/0f8ade4bee86bab84fa65c9de1294f200ada0db5/corehq/apps/accounting/migrations/0046_new_plans.py#L26-L27
I'll add a cache clear to that command actually, because it would be confusing for that to take a day to kick in.